### PR TITLE
Vumi Bridge transport can't access Vumi Go over HTTPS

### PR DESCRIPTION
**Pull request by hodgestar**
Vumi Go's SSL certificate relies on a weak (SHA1) intermediate certificate. This fails to be verified by a lot of server side software (including Twisted) but is accepted by current browsers. Vumi Go's certificate will be updated in October at the latest, but until then the bridge transport needs to accept weaker certificates.

### CODE REVIEW

**Comment by hodgestar** [comment]
@JayH5 @imsickofmaps Ready for review.

File: vumi/transports/vumi_bridge/tests/test_vumi_bridge.py [hunk 1]
```diff
@@ -178,6 +177,11 @@ def test_receive_bad_event(self):
         self.assertEquals(status['message'], 'Bad event received from Vumi Go')
 
     @inlineCallbacks
+    def test_weak_cacerts_installed(self):
+        yield self.get_configured_transport()
+        self.assertEqual(os.environ["SSL_CERT_FILE"], certifi.old_where())
```
Comment (hodgestar) [inline vumi/transports/vumi_bridge/tests/test_vumi_bridge.py]: Not sure if there is a clean way to write a better test for this.

**Comment by hodgestar** [comment]
And @jerith ^^^

**Comment by hodgestar** [comment]
@morgs Since this involves SSL, I thought I'd ask you whether you think it's okay to use the weaker CA certificates for now?

**Comment by jerith** [comment]
:+1: although we should test it against some real HTTPS servers before releasing.

**Pull request merged**